### PR TITLE
Get email address for the power bi view activity

### PR DESCRIPTION
**Pull request by elic-eon**
### 🤔 Why?

We need to show the actors even if they are not in Metaphor system. If we only captured `personId` we would not be able to get unknown person information.

### 🤓 What?

Fill email in actorInfo

### 🧪 Tested?

Unit test

### CODE REVIEW

File: metaphor/power_bi/extractor.py [hunk 1]
```diff
@@ -690,6 +693,7 @@ def extract_activities(self) -> List[UserActivity]:
                     actor=user_id,
                     measure=1.0,
                     timestamp=activity.CreationTime,
+                    actor_info=UserActivityActorInfo(email=activity.UserId),
```
Comment (usefulalgorithm) [inline metaphor/power_bi/extractor.py:696]:
```suggestion
                    actor_info=UserActivityActorInfo(email=actor_email),
```

**Review by usefulalgorithm — CHANGES_REQUESTED** [review]
just a nit, otherwise lgtm

**Review by usefulalgorithm — APPROVED** [review]
LGTM

File: tz.txt [hunk 2]
```diff
@@ -0,0 +1,19 @@
+localTimeZoneId='Dateline Standard Time'
```
Comment (mars-lan) [inline tz.txt:1]: Where is this used?

Comment (elic-eon) [reply tz.txt]: Dropped

**Review by mars-lan — APPROVED** [review]

**Pull request merged**